metrics-core/pom.xml

```diff
@@ -51,6 +51,30 @@
       <version>${trove4j.version}</version>
     </dependency>
 
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-all</artifactId>
+      <version>4.0.23.Final</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.apache.cassandra</groupId>
+      <artifactId>cassandra-clientutil</artifactId>
+      <version>2.1.1</version>
+    </dependency>
+
+    <dependency>
```

Comment:
I didn't notice this before. Why does metrics-core have cassandra server dependencies now? And the version of Netty should be the same as the one used by the DataStax driver.